README.md

```diff
@@ -34,68 +51,217 @@ The project consists of four components:
   that come with translation server. This makes it easier to test script
   problems and allows you to deploy updates as they happen in Github.
 
-## Deployment
+## Installation
 
 There are only two hard requirements for getting zts-in-a-box up and running:
 
 * [Docker](https://docs.docker.com/engine/installation/)
 * [Docker Compose](https://docs.docker.com/compose/install/)
 
-Once you have them installed, clone this repository and build the containers:
+Once you have them installed, clone this repository and build/pull the containers:
 
 ```sh
 git clone --recursive https://github.com/kba/zts-in-a-box
 cd zts-in-a-box
-docker-compose up
+docker-compose pull
+docker-compose build
 ```
```

Comment:
It would be nice to mention already here, what one has to do additionally under Windows/Mac, such that I can read from top to bottom and perform the necessary steps one after the other (without going forward and backward).